video-sync-frontend/src/index.css

```
@@ -1,190 +1,109 @@
+   color on every element at once, which is where the old UI's syrupy
+   feel came from. Components own their own transitions. */
+
+html {
+  background: #0f0d0a;
+}
+
 body {
-  font-family: 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Oxygen',
-    'Ubuntu', 'Cantarell', 'Fira Sans', 'Droid Sans', 'Helvetica Neue',
-    sans-serif;
+  font-family: 'Inter', system-ui, -apple-system, sans-serif;
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)